HACKING.md

```diff
@@ -19,27 +19,37 @@ recommended to do that:
  - It avoids having to install development packages on your main machine.
  - It avoids having to map the build and test dependencies to package names of various distributions.
 
-On Fedora/CentOS/RHEL based distributions, install toolbox with
+1. Install `toolbox`
 
-    sudo dnf install toolbox
+   - Fedora/CentOS/RHEL based distributions:
 
-On Debian/Ubuntu based distributions, install it with
+         sudo dnf install toolbox
 
-    sudo apt install podman-toolbox
+   - Debian/Ubuntu based distributions:
 
-After that, create a cockpit development toolbox:
+         sudo apt install podman-toolbox
 
-    toolbox create --image quay.io/cockpit/tasks -c cockpit
+2. Create a development toolbox for Cockpit
 
-Enter the toolbox with
+       toolbox create --image quay.io/cockpit/tasks -c cockpit
 
-    toolbox enter cockpit
+3. Enter the toolbox:
+
+       toolbox enter cockpit
 
 Your home directory, user D-Bus, etc. are shared with the host, so you can
 edit files as you normally would. Building and running tests happens inside the
 toolbox container. If desired, you can install additional packages with
 `sudo dnf install`.
 
+The Cockpit team occasionally refreshes that container. To re-create yours from
+a new image, run
```

Comment:
"That" is weird. It should either refer to the specific container image (as it's talking about the local container) or at least "this" instead of "that" (however, then it's ambiguous.

Preferred solution:

```suggestion
The Cockpit team occasionally refreshes the `tasks` container image. 
To re-create your development container from the latest image, run:
```

I've also added a colon to be consistent with command prefixing that's above.